src/plugins/kibana/lib/keys_deep.js

```diff
@@ -0,0 +1,23 @@
+const _ = require('lodash');
+
+export default function keysDeep(object, base) {
+  let result = [];
+  let delimitedBase = base ? base + '.' : '';
+
+  _.forIn(object, (value, key) => {
```

Comment:
Just a sanity check, are you sure you want to use `forIn` instead of `forEach`? `forIn` will also iterate over prototype properties.